code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -26,6 +26,7 @@
 	var/cpr_time = 1
 	nutrition = 400//Carbon
 
+	var/is_watching = 0  //used for remote viewing of multiz structures
```

Comment:
TRUE and FALSE would work nicer for this, but eh who cares

Reply:
I care. Please use TRUE and FALSE.

Reply:
done